### PR TITLE
Porting of Relu variant to TFLite Micro.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
This is for HiKey Borads.

### CODE REVIEW

**Comment by amitsrivastava78** [comment]
I am planning to put padding, round, prelu  along with lstm support for tflite Macro, let me know if this work is conflicting with anyone.

Regrds
Amit

**Comment by amitsrivastava78** [comment]
@aselle , I have resolved the merge conflicts, can you please take a look.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@petewarden , great fan of your blogs , i have re-based the code again ,kindly have a look, also let me know if you want me to split the activation kernels into different files ?

Regards
Amit

**Comment by amitsrivastava78** [comment]
@petewarden  can you please respond to the above query.

Regards
Amit

**Comment by gbaned** [comment]
@amitsrivastava78 Could you please resolve the conflicts? Thanks!

**Comment by amitsrivastava78** [comment]
> @amitsrivastava78 Could you please resolve the conflicts? Thanks!

@gbaned , conflict resolved.

Regards
Amit

File: tensorflow/lite/experimental/micro/kernels/relu_variants.cc [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  int diff_min = 0;
+};
+
+struct PreluOpData : public OpData {
+  int32_t output_multiplier = 0;
+  int output_shift = 0;
+};
+
+void* PreluInit(TfLiteContext* context, const char* buffer, size_t length) {
+  return new PreluOpData;
```
Comment (petewarden) [inline tensorflow/lite/experimental/micro/kernels/relu_variants.cc]: We can't use new/delete in Micro unfortunately. You can check out some of the other kernel implementations for the pattern we use instead.

**Review by petewarden — CHANGES_REQUESTED** [review]

**Comment by amitsrivastava78** [comment]
@petewarden , thanks for your comments, i have updated the code as per your suggestion, 
Resolved the merge conflicts and tested the code again. kindly have a look.

Regards
Amit

**Comment by gbaned** [comment]
@amitsrivastava78 Could you please resolve the conflicts? Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**